### ISSUE TITLE
RP2350: Add legacy PIO dvi mode in picodvi (framebuffer_RP2350)

### ISSUE THREAD

**Issue by RetiredWizard** [adafruit/circuitpython#9588]
I wanted to try using a Pico2 with a piCowbell DVI or a Pimoroni Pico DV base so I built a Pico2 firmware with picodvi enabled. I'm guessing that the Framebuffer_RP2350 routine has been updated to support HSTX and dedicated(?) pins associated with that peripheral since it kicks out the PIO pins used by the older Pico DV devices as invalid.

This may only be a short term issue until new HSTX versions of these devices become available but it would be nice if there was some legacy support to run pio DVI devices using the new RP2350 boards.

**Comment by tannewt** [comment]
Yes, `picodvi` is HSTX specific on RP2350. For Pico you can use: https://learn.adafruit.com/adafruit-dvi-sock-for-pico
